javascript/node/selenium-webdriver/safari.js

```diff
@@ -179,6 +196,23 @@ class Options {
   }
 }
 
+/**
+ * @param  {(Options|Object<string, *>)=} o The options object
+ * @return {boolean}
+ */
+function useTechnologyPreview(o) {
+  if (o instanceof Options) {
+    return !!o.options_[TECHNOLOGY_PREVIEW_OPTIONS_KEY];
```

Comment:
`options_` may be null. Need:

`!!(o.options_ && o.options_[TECHNOLOGY_PREVIEW_OPTIONS_KEY]);`